UnityProject/Assets/Prefabs/Objects/Electricity/PowerGeneratorPlasma.prefab

```diff
@@ -5332,9 +5332,9 @@ MonoBehaviour:
   Previouscurrent: 0
   current: 0
   PreviousSupplyingVoltage: 760000
-  SupplyingVoltage: 760000
+  SupplyingVoltage: 500000
   PreviousInternalResistance: 76000
-  InternalResistance: 76000
+  InternalResistance: 50000
```

Comment:
What are the implications of reducing this to 500 kV? Does the PACMAN still power the stations?

Also, reducing the internal resistance I imagine would buff the current supply of the PACMAN. Is that intended?

Reply:
Lowered values just for testing, forgot to set em back. Fixed in new commit